ui/webpack.common.js

```diff
@@ -0,0 +1,48 @@
+const autoprefixer = require('autoprefixer');
```

Comment:
I believe you need copyright & license header for every file.

Reply:
Sent you an email for details, but so far I believe the short answer is yes. But feel free to submit it without license header for now. We can attach them once we conclude that we really need them.

Reply:
Done.